manifests/a/Amazon/Games/latest/Amazon.Games.locale.en-US.yaml

```diff
@@ -0,0 +1,26 @@
+# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+
+PackageIdentifier: Amazon.Games
+PackageVersion: latest
+PackageLocale: en-US
+Publisher: Amazon.com, Inc.
```

Comment:
```suggestion
Publisher: Amazon.com Services, Inc.
```

Reply:
I can change, but it's not from Amazon Services. I got the information from the app about tab.

Reply:
Publisher needs to match Control Panel because Windows Package Manager uses this information for `winget list` and `winget upgrade`.

Reply:
Good to know, I'll take this information from the control panel for the next manifests